web2.0/src/Components/Current/CurrSwell.js

```diff
@@ -0,0 +1,72 @@
+import { useState, useEffect } from 'react';
+import WeatherCard from '../weather-card/WeatherCard';
+
+const CurrSwell = () => {
+  const [currentData, setCurrentData] = useState();
+
+  const getData = async () => {
+    const response = await fetch('http://localhost:8000/swell');
```

Comment:
Always useful to refactor hardcoded strings, such as all your URLs that you fetch from, to constants at minimum. 

If you're up for it, we should practice building and running your code in production on a server (e.g., GitHub Pages, which is free) so you can see the  benefits of such refactors.

When deploying your code, you may not be fetching from `localhost`, but rather another domain (or even ip-address). Such refactors will save you headache and time when working with both.
I recommend practicing deployment when you're ready, so you can see